src/DependencyInjection/SurfnetStepupExtension.php

```diff
@@ -107,19 +111,49 @@ public function load(array $config, ContainerBuilder $container)
                     $config['locale_cookie']['http_only'],
                 ]
             );
+    }
 
-        $container->getDefinition('surfnet_stepup.form.choice_list.locales')
-            ->replaceArgument(0, $container->getParameter('locales'));
+    /**
+     * @param array            $config
+     * @param ContainerBuilder $container
+     */
+    private function configureGatewayApiClient(array $config, ContainerBuilder $container)
+    {
+# Configure the Gateway API SMS service's Guzzle client.
```

Comment:
Nit: This was probably in the original code as well, but wouldn't this look nicer with a splash of indentation?

Aside: I haven't seen `#`-style comments that often, are they allowed in this project for single line comments?